### ISSUE TITLE
Add a metrics logging mechanism

### ISSUE THREAD

**Issue by scanon** [NERSC/podman-hpc#115]
We need a mechanism to gather metrics to report things like what images are being used, unique users, jobs sizes, etc.

**Comment by brandongc** [comment]
We could probably work up a straightforward nerscjson_client integration. We already have python and lua bindings.

**Comment by dingp** [comment]
Yes, using nersc_json’s python binding is exactly what’s in the plan. I have a few other things to wrap up but should be able to get onto this in the next week or two.
